pb/custom.go

```diff
@@ -0,0 +1,89 @@
+package pstore_pb
+
+import (
+	"encoding/json"
+
+	peer "github.com/libp2p/go-libp2p-peer"
+	pt "github.com/libp2p/go-libp2p-peer/test"
+	ma "github.com/multiformats/go-multiaddr"
+)
+
+type ProtoPeerID struct {
+	peer.ID
+}
+
+func NewPopulatedProtoPeerID(r randyPstore) *ProtoPeerID {
```

Comment:
Yes, they are required by the custom types gogo plugin. gogo generates `NewPopulated` functions for testing the types it controls, but you need to manually create them for the custom types. Moved them to `custom_test.go`, although this breaks the symmetry with Gogo which puts them in the `.pb.go` instead of in the `_test.go`, I think it's clearer in our case.

Reply:
Actually, _no can do_. The gogo-generated protobuf code in `pstore.pb.go` calls these functions, and putting them in a `_test.go` hides them. Will keep them where they are, and comment them appropriately.